### ISSUE TITLE
`make` annoyingly stops to ask about moving files

### ISSUE THREAD

**Issue by fiatjaf** [ElementsProject/lightning#5567]
This is not really a bug, but it's annoying that we have to watch the compilation process attentively otherwise it might get stuck while waiting for an answer.

If we do `make` on the main repo, sometimes during the compilation it will stop and ask things like this:

```
mv: replace 'tools/headerversions', overriding mode 0755 (rwxr-xr-x)?
```

**Comment by SimonVrouwe** [comment]
Prepending `make clean` (or `make distclean`) helps?

**Comment by fiatjaf** [comment]
No.

**Comment by vincenzopalazzo** [comment]
> I had tried make clean with no success.

there is a way to reproduce this? maybe I need to write a clean cloud machine?

**Comment by fiatjaf** [comment]
It has happened to me in two different machines where I just `git pull`, `./configure` and `make` -- then some days later `git pull` and `make` and so on.

**Comment by rustyrussell** [comment]
My guess is that you did a "sudo make" at some point.  Now root owns those files, and so mv is asking if you want to override them?

**Comment by fiatjaf** [comment]
Yes, I did, `sudo make install`.

But should I be able to mv over root-owned files? How do I recover from this? `make clean` didn't work. I guess `sudo make clean`?

Let me bring over another issue, then: I did `sudo make install` just after `make` and everything started to be compiled again, even though everything was already compiled. If that hadn't been the case I wouldn't have root-owned files, right?
